books/RayTracingInOneWeekend.html

```diff
@@ -1429,8 +1430,56 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     [Listing [rtweekend-initial]: <kbd>[rtweekend.h]</kbd> The rtweekend.h common header]
 
+For all main program files, we will include `rtweekend.h` first, so most other header files (where
+the bulk of our code will reside) can assume that they are free to assume that these definitions are
```

Comment:
> ...can assume that they are free to assume...

..._can assume that these definitions are already available._ ...

Reply:
Done.